events/filter.go

```diff
@@ -48,7 +48,7 @@ func Consume[T any](subscriber <-chan T, process func(event T)) jobs.Job {
 
 func recovery(errChan chan<- error) {
 	if r := recover(); r != nil {
-		errChan <- fmt.Errorf("job panicked:%s", r)
+		errChan <- fmt.Errorf("job panicked: %s", r)
```

Comment:
take this out of this PR, it's already being touched in #55

Reply:
yep, fixed